Fix multiline formatting bug (#11)
st--/annotate-equations: annotate-equations.tex

\subsection{Default formatting of annotation labels}

\verb|\eqnannotationfont| is a one-argument command that feeds into the

> \texttt{font} field of tikz and can be used to have consistent formatting:

Reviewer: ```suggestion
\texttt{font} field of TikZ and can be used to have consistent formatting:
```

> \verb|\eqnannotationfont| is a one-argument command that feeds into the
> \texttt{font} field of TikZ and can be used to have consistent formatting:

Reviewer: ```suggestion
\verb|\eqnannotationfont| sets the \texttt{font} field of the TikZ annotation label and can be re-set to change its formatting:
```

\vspace{1em}
\end{LTXexample}
\noindent

\verb|\eqnannotationstrut| is defined to be a strut (zero-width height) to
provide minimum distance between the text and the corresponding arrow line. By

> default it is \verb|\strut|, which has, which has a similar effect to

Reviewer: ```suggestion
default it is \verb|\strut|, which has a similar effect to
```

\vspace{1em}
\end{LTXexample}
\noindent

> \verb|\eqnannotationstrut| is defined to be a strut to provide minimum distance
> from text to the surrounding diagrams. By default it is \verb|\strut|, which has

Reviewer: ```suggestion
\verb|\eqnannotationstrut| is defined to be a strut (zero-width height) to provide minimum distance between the text and the corresponding arrow line. By default it is \verb|\strut|, which has
```

a similar effect to \verb|\mathstrut| in \verb|\eqnhighlightheight|.

\begin{LTXexample}[text outside listing,lefthand width=0.5in]
% Default

> \renewcommand{\eqnannotationstrut}{\strut}

Reviewer: ```suggestion
\renewcommand{\eqnannotationstrut}{\strut}  % package default
```
for consistency

\begin{equation*}

> 	\eqnmarkbox[blue]{v}{v}
> \end{equation*}
> \annotate[yshift=-0.5em]{below}{v}{velocity}

Reviewer: ```suggestion
    \eqnmarkbox[blue]{size}{s}
\end{equation*}
\annotate[yshift=-0.5em]{below}{size}{the size}
```
I think the effect is more striking with text that doesn't have descenders (like in `y`).
also, changed the tab to spaces for consistency with the rest of the formatting.

\vspace{1em}
\end{LTXexample}
\noindent
%
\begin{LTXexample}[text outside listing,lefthand width=0.5in]
	\renewcommand{\eqnannotationstrut}{}

	\begin{equation*}
    \eqnmarkbox[blue]{v}{v}
	\end{equation*}
	\annotate[yshift=-0.5em]{below}{v}{velocity}

> 	\vspace{1em}

Reviewer: ```suggestion
\renewcommand{\eqnannotationstrut}{}
\begin{equation*}
    \eqnmarkbox[blue]{size}{s}
\end{equation*}
\annotate[yshift=-0.5em]{below}{size}{the size}
\vspace{1em}
```
in line with above

\end{LTXexample}

> \noindent

Reviewer: I don't think this is necessary here (at the end of a section)?
```suggestion
\noindent
```

\end{LTXexample}
\noindent
%

> Here is a manual work-around: (obsolete)

Reviewer: I don't think it's helpful to show the old work-around - could you just delete it instead? All it needs to say here is that using `\\` within the text to get a line break works just fine.:)

\begin{itemize}
    \item \texttt{label above}/\texttt{label below} is not implemented for \verb|\annotate|.

>     \item Formatting only covers first line in multi-line annotation texts (see \cref{sec:multiline}).

Reviewer: Thanks for fixing it :tada:
